### PR TITLE
Pdo support

### PR DESCRIPTION
**Pull request by Zlugling**

### CODE REVIEW

File: MOMCompound.class.php [hunk 1]
```diff
@@ -65,7 +69,13 @@ public function save($metaData = null)
 		if (($row = self::getRowByIds($ids)) === NULL)
 			throw new BaseException(BaseException::OBJECT_NOT_UPDATED, get_called_class().'->'.__FUNCTION__.' failed to update object with data from database');
 
+		$isNew = $this->__mbNewObject;
 		$this->fillByObject($row);
+
+		if ($isNew)
```
Comment (gyde) [inline MOMCompound.class.php]: Brug $this->isNew()

File: MOMSimple.class.php [hunk 2]
```diff
@@ -129,13 +129,14 @@ public function save($metaData = null)
 		if (($row = self::getRowById($id)) === false)
 			throw new BaseException(BaseException::OBJECT_NOT_UPDATED, get_called_class().'->'.__FUNCTION__.' failed to update object with metadata from database');
 
+		$isNew = $this->__mbNewObject;
 		$this->fillByObject($row);
 
-		$selector = self::getSelector($id);
-		if ($this->__mbNewObject)
-			static::setStaticEntry($selector, $this);
+		if ($isNew){
```
Comment (gyde) [inline MOMSimple.class.php]: Brug $this->isNew()

Comment (Zlugling) [reply MOMSimple.class.php]: `$this->isNew()` uses `$this->__mbNewObject` which gets overwritten by the preceding call to `$this->fillByObject()` (which is why the test failed earlier, as we discussed)

I could of course use it on line 132, changing it to `$isNew = $this->isNew();` but that would stand in contrast to line 122 where `$this->__mbNewObject` is used directly

Comment (gyde) [reply MOMSimple.class.php]: Yes, makes sense. Just rename the variable to make it more obvious that it's saving the previous state.
Maybe $previousNewState = $this->isNew()

**Pull request merged**